src/main/scala/Server.scala

```diff
@@ -102,46 +70,44 @@ object Server extends App {
         }
 
       case Failure(error: SyntaxError) ⇒
-        complete(BadRequest, JsObject(
+        complete(ToResponseMarshallable(BadRequest → JsObject(
```

Comment:
Not sure if calling ToResponseMarshallable is necessary. I made the change just to make the compiler happy.